src/test/java/com/glencoesoftware/omero/zarr/ZarrPixelBufferTest.java

```diff
@@ -246,11 +210,9 @@ public void testGetChunks() throws IOException {
         int sizeZ = 1;
         int sizeY = 512;
         int sizeX = 2048;
-        int resolutions = 3;
```

Comment:
Here and elsewhere in this class, might be worth keeping the `resolutions` variable and referencing `resolutions` instead of hard-coding 3 everywhere. That's a little easier to understand, and would make it easy to adjust the resolution count if we wanted to later on.